components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Twin/src/Deploy/IoTHubSupervisorDeployment.cs

```diff
@@ -103,20 +104,19 @@ public sealed class IoTHubSupervisorDeployment : IHostProcess {
                 });
             }
             else {
-                // Windows
+                // Eflow
```

Comment:
I guess since eflow configuration is identical the Linux one, so we can simplify the code here.

Reply:
I had it simplified, but /mount is readonly on eflow. I dont want to break existing linux deployments.